charts/zitadel/templates/setupjob.yaml

```diff
@@ -79,8 +79,25 @@ spec:
             mountPath: /config
           - name: chowned-secrets
             mountPath: /.secrets
+          {{- if and (not (empty .Values.zitadel.configmapConfig.FirstInstance)) (and (not (empty .Values.zitadel.configmapConfig.FirstInstance.Org)) (not (empty .Values.zitadel.configmapConfig.FirstInstance.Org.Machine))) }}
+          - name: machinekey
+            mountPath: {{ dir .Values.zitadel.configmapConfig.FirstInstance.MachineKeyPath }}
+          {{- end}}
           resources:
             {{- toYaml .Values.setupJob.resources | nindent 14 }}
+        {{- if and (not (empty .Values.zitadel.configmapConfig.FirstInstance)) (and (not (empty .Values.zitadel.configmapConfig.FirstInstance.Org)) (not (empty .Values.zitadel.configmapConfig.FirstInstance.Org.Machine))) }}
+        - name: "{{ .Chart.Name}}-machinekey"
+          image: alpine/k8s:1.25.2
```

Comment:
can we use the kubeapi's k8s version?

Reply:
is the information what k8s-version is used already in the chart?

Reply:
added through build-in helm object "Capabilities.KubeVersion"